ui/src/app/table/select-all/select-all.component.scss

```diff
@@ -0,0 +1,8 @@
+select {
+  width: fit-content;
+  font-weight: bolder;
+  font-family: 'Open Sans',sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  -webkit-appearance: menulist;
+}
```

Comment:
As per Paola's and John's feedback, we should add a border to the select like the items per page border select

Reply:
To reuse the style now there is a CSS class ".select" in styles.scss that is used in size component and select all component, and I changed the file extension of the file.